.circleci/config.yml

```diff
@@ -0,0 +1,117 @@
+# Configuration file for https://circleci.com/gh/angular/dev-infra
+
+version: 2.1
+
+# **Note**: When updating the beginning of the cache key, also update the cache key to match
+# the new cache key prefix. This allows us to take advantage of CircleCI's fallback caching.
+# Read more here: https://circleci.com/docs/2.0/caching/#restoring-cache.
+cache_key: &cache_key v7-{{ checksum ".bazelversion" }}-{{ checksum "WORKSPACE" }}-{{ checksum "yarn.lock" }}
+# We want to invalidate the cache if the postinstall patches change. In order to apply new
+# patches, a clean version of the node modules is needed. Additionally, we invalidate the cache
+# if the Bazel version changes. We do this because otherwise the `bazelisk` cache folder will
+# contain all previously used versions and ultimately cause the cache restoring to be slower.
+cache_fallback_key: &cache_fallback_key v7-{{ checksum ".bazelversion" }}-
+
+gcp_decrypt_token: &gcp_decrypt_token "angular"
+
+restore_cache: &restore_cache
```

Comment:
FWIW: I prefer the way do this in the comp repo where we name it `var_1: &<name>`. This avoids the ambiguity where you could think that `restore_cache` is an actual config setting from CircleCI here.